src/data/StreamDetails.cpp

```diff
@@ -219,8 +219,8 @@ void StreamDetails::loadWithLibrary()
         if (lang.isEmpty()) {
             lang = MI2QString(mi.Get(Stream_Audio, i, QString2MI("Language/String")));
         }
-        QString audioCodec = audioFormat(MI2QString(mi.Get(Stream_Audio, i, QString2MI("Codec"))),
-            MI2QString(mi.Get(Stream_Audio, i, QString2MI("Format_Profile"))));
+        QString audioCodec = audioFormat(MI2QString(mi.Get(Stream_Audio, i, QString2MI("Format"))),
+            MI2QString(mi.Get(Stream_Audio, i, QString2MI("CodecID"))));
```

Comment:
What MediaInfo version did you use? I don't know which tag it was but some `Format_*` is deprecated. I'm find with it if it works, though :)

Reply:
v19.09

Reply:
Good. :+1: